shell/client/styles/_topbar.scss

```diff
@@ -62,6 +62,56 @@ body>.topbar-update {
   }
 }
 
+body>li.navitem-grain {
```

Comment:
Ahhh, missed the note below your screenshot in the PR cover message.  Never mind.